src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -39,6 +39,46 @@
                     "parameters": []
                 }
             ]
+        },
+        {
+            "path": "/v1/debug/peer_status/{id}",
+            "operations": [
+                {
+                    "method": "GET",
+                    "summary": "Get peer status",
+                    "type": "peer_status",
+                    "nickname": "get_peer_status",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "parameters": [
+                        {
+                            "name": "id",
+                            "in": "path",
+                            "required": true,
+                            "type": "long"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "path": "/v1/debug/partition_leaders_table_2",
```

Comment:
is that required ?

Reply:
No. It's leftover from debugging, Will remove.

Reply:
Done.